### ISSUE TITLE
The accuracy of {net:ZF, train-set: 07trainval, test-set: 07test} is higher than paper

### ISSUE THREAD

**Issue by zhonhel** [rbgirshick/py-faster-rcnn#757]
The paper is 59.9.
My running result is 60.9 with no modification on this repo.
The training method is end2end.
So, I want to know why higher than paper....

Can anyone help me? Thank you very much~

**Comment by MdAshrafulAlam** [comment]
I am trying to run the demo.py. but, when I try this command " /data/scripts/fetch_faster_rcnn_models.sh ", then it shows these errors.
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
Could anyone please explain what is the solution? I am really in trouble and very new in deep learning. It would matter of glad if you help me.

**Comment by zhonhel** [comment]
You can directly download faster_rcnn_models by this URL:
https://dl.dropboxusercontent.com/s/o6ii098bu51d139/faster_rcnn_models.tgz?dl=0

**Comment by zhonhel** [comment]
@MdAshrafulAlam 
And you can check whether you have already downloaded faster_rcnn_models.tgz, it is 694.81MB.

**Comment by MdAshrafulAlam** [comment]
yeah, I used this link before but the tar file cannot be downloaded fully.

**Comment by zhonhel** [comment]
@MdAshrafulAlam 
Maybe use a better downloader is better : )

**Comment by MdAshrafulAlam** [comment]
Can I contact with you personally?

**Comment by liukuixiang** [comment]
@zhonhel 
do you training vgg16 model get mAP is higher than the paper?

**Comment by zhonhel** [comment]
@liukuixiang
I didn't train vgg16 on pascal voc dataset..

**Comment by lixiangamy** [comment]
how to get train accuracy?
